aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -726,7 +726,11 @@ Tensor& add_out_dense_sparse_cpu(Tensor& r, const Tensor& dense, const SparseTen
 
 Tensor mul_sparse(const Tensor& self, const Tensor& other) {
   auto commonDtype = at::result_type(self, other);
-  Tensor result = at::empty({0}, self.options().dtype(commonDtype));
+  // This code path is hit when doing `Tensor(2.) * other`, which will actually
```

Comment:
You could be a bit more general here, please?

Abstractly describe what the usual behavior is and why it needs to be different for sparse.

Reply:
done. The usual behaviour is described on line 724, should I repeat it here?